File: Java/BasicAlgorithmPractice/src/ArrayOfNumbers.java
public class ArrayOfNumbers {
    private int[] numbers;

    public ArrayOfNumbers(int[] numbers) {
        this.numbers = numbers;
    }

    public int getHighestNumber() {
        int highestNumber = 0;
        for (int number : this.numbers) {
            if (number > highestNumber) {
                highestNumber = number;
            }
        }
        return highestNumber;
    }

    public int[] getNumbers() {
        return numbers;
    }
}

File: Kotlin/BasicAlgorithms/src/main/kotlin/numbers/ArrayOfNumbers.kt
package numbers

class ArrayOfNumbers(val numbers: IntArray) {

    fun getHighestNumber():Int {
        var highestNumber = 0;
        numbers.forEach { number ->
            if(number > highestNumber) {
                highestNumber = number;
            }
        }
        return highestNumber;
    }
}